src/allotropy/allotrope/schema_mappers/adm/liquid_chromatography/benchling/_2023/_09/liquid_chromatography.py

```diff
@@ -0,0 +1,389 @@
+from dataclasses import dataclass
+from typing import Protocol, TypeVar
+
+from allotropy.allotrope.models.adm.liquid_chromatography.benchling._2023._09.liquid_chromatography import (
+    ChromatogramDataCube,
+    ChromatographyColumnDocument,
+    DerivedColumnPressureDataCube,
+    DeviceControlAggregateDocument,
+    DeviceControlDocumentItem,
+    DeviceSystemDocument,
+    InjectionDocument,
+    LiquidChromatographyAggregateDocument,
+    LiquidChromatographyDocumentItem,
+    MeasurementAggregateDocument,
+    MeasurementDocument,
+    Model,
+    PostColumnPressureDataCube,
+    PreColumnPressureDataCube,
+    ProcessedDataAggregateDocument,
+    ProcessedDataDocumentItem,
+    SampleDocument,
+    SampleFlowRateDataCube,
+    SamplePressureDataCube,
+    SolventConcentrationDataCube,
+    SystemFlowRateDataCube,
+    SystemPressureDataCube,
+    TemperatureProfileDataCube,
+)
+from allotropy.allotrope.models.shared.definitions.custom import (
+    TQuantityValueCubicMillimeter,
+    TQuantityValueMicrometer,
+    TQuantityValueMillimeter,
+)
+from allotropy.allotrope.models.shared.definitions.definitions import (
+    FieldComponentDatatype,
+    TDatacube,
+    TDatacubeComponent,
+    TDatacubeData,
+    TDatacubeStructure,
+)
+from allotropy.allotrope.schema_mappers.schema_mapper import SchemaMapper
+
+
+@dataclass(frozen=True)
+class Metadata:
+    asset_management_id: str
+    product_manufacturer: str
+    device_id: str
+    firmware_version: str
+    analyst: str
+
+
+@dataclass(frozen=True)
+class DataCubeComponent:
+    type_: FieldComponentDatatype
+    concept: str
+    unit: str
+
+
+@dataclass(frozen=True)
+class DataCube:
+    label: str
+    structure_dimensions: list[DataCubeComponent]
+    structure_measures: list[DataCubeComponent]
+    dimensions: list[tuple[float, ...]]
+    measures: list[tuple[float | None, ...]]
+
+
+@dataclass(frozen=True)
+class ProcessedDataDoc:
+    chromatogram_data_cube: DataCube | None = None
+    derived_column_pressure_data_cube: DataCube | None = None
+
+
+@dataclass(frozen=True)
+class DeviceControlDoc:
```

Comment:
DeviceControlDoc can also be merged into measurement document (note that while device control doc can be a list, we never really use it as such - all of your cases - and all cases in allotropy - use a single value)

Reply:
Nevermind on this - I see you do have one example that has 2 device control documents.